src/app/api/admin/users/route.ts

```
@@ -0,0 +1,112 @@
+import { NextResponse } from 'next/server';
+import { prisma } from '@/lib/prisma';
+import { getCurrentUser } from '@/lib/session';
+import { Role } from '@prisma/client';
+import { Role, UserStatus } from '@prisma/client';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Role'. Consider to delete it or rename it.

'Role' is defined here:


(lint/suspicious/noRedeclare)